server/utils/utils.ts

```diff
@@ -5,8 +5,10 @@ const properCase = (word: string): string =>
 
 const isBlank = (str: string): boolean => !str || /^\s*$/.test(str)
 
+export const getDateFromAPI = (dateString: string): Date => parse(dateString, 'yyyy-MM-dd', new Date())
```

Comment:
Created as we're likely to need to convert a date from endpoints quite frequently.